packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1890,10 +1898,24 @@ export class ContainerRuntime
 			"Loaded",
 			undefined /* timestampMs */,
 			dataStoreContext.packagePath,
```

Comment:
It's not obvious to me looking at the different implementations of `getInitialSnapshotDetails` that this is guaranteed not to assert.  Thoughts?

Reply:
Changed it to get package path from the initial snapshot details.